domain.yml

```diff
@@ -427,6 +428,8 @@ templates:
     - text: "I can help you solving commonon problems with Rasa 🙂"
   utter_get_started_step3:
     - text: "I can help you to install Rasa Stack."
+  utter_ask_faq_tutorials:
+    - text: "Yes, [here](https://blog.rasa.com/tag/tutorials/) are some tutorial I wrote, but you can also check out some great tutorials written by the Rasa community [here](https://docs.google.com/spreadsheets/d/1lzvSRsTr0mQdJT_usvo1jMDzHjzvjVbiUkHpiGIKUgA/edit#gid=1717872446)"
```

Comment:
Should we maybe give a list of tutorial links until the docs page is finished?

Reply:
There are too many.
I want to try and do the docs this week (though I'm not sure how much time that takes).
But if we want to deploy this before the docs are finished I suggest we just leave out the community tutorials for now and just add them once the docs are done.

Reply:
I just made that change

Reply:
yes that sounds good @imLew